Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -2036,6 +2306,9 @@ eval (GenderBannerModal (Banner.OnClick)) state = exit $ GoToMyProfile state tru
 
 eval _ state = continue state
 
+dummyChatComponent :: MessagingView.ChatComponent
```

Comment:
use the dummy component declared in screenData instead of creating another function.

Reply:
There is cyclic dependency issue